main.go

```diff
@@ -65,6 +62,7 @@ func init() {
 	utilruntime.Must(v1.AddToScheme(eventsScheme))
 	utilruntime.Must(policiesv1.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
```

Comment:
Instead of this log message, could you please just move `//+kubebuilder:scaffold:scheme` to just above `utilruntime.Must(policiesv1.AddToScheme(scheme))`?